Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift

```diff
@@ -110,6 +110,32 @@ final class SentryContinuousProfilerTests: XCTestCase {
         manualSpan.finish()
         automaticSpan.finish()
     }
+
+    // test that when stop is called, the profiler runs to the end of the last
+    // chunk and transmits that before stopping
+    func testStoppingProfilerTransmitsLastFullChunk() throws {
```

Comment:
Thanks for the comment. We need to properly document this behavior for the `SentrySDK.stopProfiler` method once we make it public cause this could confuse users quite a bit.

Reply:
Ah yeah I'll add that to this PR.